### PR TITLE
feature2 added

### PR DESCRIPTION
**Pull request by gom20**
pull request

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -8,3 +8,4 @@ console.log('working in feature/TEST01');
```
Comment (HyungukYoo) [inline main.js]:
```suggestion
console.log('test1')
```

File: feature2.js [hunk 2]
```diff
@@ -0,0 +1 @@
+console.log('test');
```
Comment (HyungukYoo) [inline feature2.js:1]:
```suggestion

```

File: feature2.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+
+console.log('modified');
```
Comment (HyungukYoo) [inline feature2.js]:
```suggestion
console.log('modified2');
```

**Review by HyungukYoo — COMMENTED** [review]
See my review

**Review by gom20 — COMMENTED** [review]
commit

**Review by HyungukYoo — APPROVED** [review]
LGTM

**Pull request merged**